gdk/telemetry/emit.py

```diff
@@ -9,8 +11,16 @@ class Emit:
     """
 
     def __init__(self, emiter: ITelemetry = None):
+        self.runtime_config = RuntimeConfig()
         self._emiter = emiter or Telemetry()
 
     def installed_metric(self):
+        """
+        Sends an installed metric only once after the cli has been installed
+        """
+        if self.runtime_config.has(ConfigKey.INSTALLED):
```

Comment:
check for this version specifically?

Reply:
Yeah I want to write a test on the next PR so we emit it if the version has changed

Reply:
Actually I will just add it one this one